### DISCUSSION TITLE
What numbers are what difficulty for "dota_bot_set_difficulty"?

### DISCUSSION THREAD

**Discussion by Espionage724** [ValveSoftware/Dota2-Gameplay#15723] [Q&A] [answered]
https://developer.valvesoftware.com/wiki/Dota_Bot_Scripting#Bot_Difficulties says there's 6

- Passive
- Easy
- Medium
- Hard
- Unfair
- New Player

https://developer.valvesoftware.com/wiki/List_of_Dota_2_console_commands_and_variables#D says:

```
Sets all bots to a particular difficulty (0 = easy, 1 = medium, 2 = hard, 3 = unfair. 
```
But what about Passive and New Player? It seems like 0 would start at Passive and Easy being 1?

Is there a local script file that mentions the numbers? Or any way to show the bot difficulty during a match? Or any up-to-date source that mentions what numbers are what difficulty?

I want to launch Dota 2 immediately into an Unfair bot match and have this that works, and want to make sure the difficulty number is for Unfair:

```
+sv_lan 1 +dota_bot_set_difficulty 4 +dota_start_ai_game 1 +map dota
```

**Comment by Espionage724** [comment]
I'm still interested in this!

**Comment by Espionage724** [comment]
Still interested in this!

**Reply by andre1891** [reply]
Had the same Question!

**Comment by Espionage724** [comment, marked as answer]
Figured something out! At the main menu with Console enabled, start a Bot match and check `dota_bot_practice_difficulty`:

- `1` = Easy
- `2` = Medium
- `3` = Hard
- `4` = Unfair
